api/Pipfile

```diff
@@ -10,6 +10,10 @@ pylint = "*"
 django = "*"
 psycopg2-binary = "*"
 autopep8 = "*"
+djangorestframework-simplejwt = "*"
```

Comment:
Are you still using this? If not, can we remove it?

Reply:
Uninstalled.